tests/infrastructure/test_manager.py

```
@@ -0,0 +1,264 @@
+        mock_config = MagicMock()
+        mock_infra = MagicMock()
+        mock_infra.mode = "local"
+        mock_config.infrastructure = mock_infra
+        mock_config_path = Path("/fake/config.yaml")
+
+        # Track the order of calls
+        call_order = []
+
+        # Create a mock provider with tracking
+        mock_provider = AsyncMock(spec=InfrastructureProvider)
+        mock_provider.health_check.side_effect = lambda **kw: (
```

Ruff (0.14.14)

[warning] Unused lambda argument: `kw`

(ARG005)